.github/workflows/kbs-e2e-azure-vtpm.yml

```diff
@@ -54,12 +54,22 @@ jobs:
       with:
         path: ./kbs.tar.gz
 
-  e2e-test:
+  tdx-e2e-test:
     needs:
     - authorize
     - checkout-and-rebase
-    uses: ./.github/workflows/kbs-e2e.yaml
+    uses: ./.github/workflows/kbs-e2e.yml
     with:
       runs-on: '["self-hosted","azure-cvm-tdx"]'
       tee: aztdxvtpm
       tarball: kbs.tar.gz
+
+  snp-e2e-test:
```

Comment:
@mkulke btw I combined the two azure vtpm test yaml into one. Does this make sense?

Reply:
as long as both keep running in parallel, it should be fine